flow/project.py

```diff
@@ -2541,7 +2618,7 @@ def _potentially_buffered(self):
         else:
             yield
 
-    def script(self, operations, parallel=False, template='script.sh', show_template_help=False):
+    def _script(self, operations, parallel=False, template='script.sh', show_template_help=False):
```

Comment:
We don't need the underscore here, we don't plan on having even an internal function here.

Reply:
@b-butler can you please clarify? Currently `script` uses `JobOperation` instance and I think that we should deprecate `script`.

Reply:
We should, but we don't need an internal function since we will just remove both when the user facing function is removed.

Reply:
This internal function is used for the command `python project.py script`.
Removing the internal function means that are we planning to remove the command line argument, `script`?

Reply:
See the discussion on #311. The appropriate solution is to have the script CLI call the underlying `submit` API (temporarily mocking the environment as `StandardEnvironment`). In the interest of keeping PRs small, though I'd be fine with adding this underscore function in this PR and then making a follow-up PR where you refactor the CLI script call to not rely on this method anymore. Does that make sense? @b-butler let me know if I missed something here, but if not, then I think no further changes are needed in this PR.

Reply:
That is fine for now then since we have an issue.